db/data_migrate/201910091570631985_add_management_charge_total_to_one_supplier.rb

```diff
@@ -0,0 +1,20 @@
+# Zendesk ticket: https://dxw.zendesk.com/agent/tickets/10151
```

Comment:
I guess it's just because there's a lot of data, but shouldn't we backfill this value for all suppliers rather than just one? Will we end up with inconsistent available data for different suppliers?

Reply:
We should probably backfill at least the big submissions to stop them causing issues. However this should be run carefully to stop the backfill itself increasing load.

Reply:
Timur has come back with another supplier causing issues. I'd love to backfill all of them but given how unperformant the db seems to be, is this wise? (I am crap with db stuff so I don't know really)

Reply:
Querying another direction might be faster, perhaps, depending on the indexing. Let's see how long this backfill takes  I guess.

Reply:
Let's file backfilling this data as an issue somewhere, then we can look at the bad indexes and backfilling as a separate task to solving the immediate problem.